src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java

```diff
@@ -11,7 +9,11 @@
 import org.gitlab4j.api.Constants.SortOrder;
 import org.gitlab4j.api.GitLabApiForm;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Date;
+import java.util.List;
+
```

Comment:
@rnemykin Would prefer to not reorder the imports in this fashion, this library uses the Eclipse ordering pattern, which was how they were previously ordered.  The Eclipse ordering puts the Java imports first followed by the imports that belong to the library, the remaining imports are alphabetically ordered and then followed by the static imports.

Reply:
Yeah, missed that, it was IDEA optimize imports options during pushing.
fixed with Eclipse convention